source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -299,5 +305,34 @@ private void ValidateMinistryRegion(long acqFileId, short updatedRegion)
                 throw new BusinessRuleViolationException("The Ministry region has been changed, this will result in a change to the file's prefix. This requires user confirmation.");
             }
         }
+
+        private void CheckAcquisitionFileStatusChanged(PimsAcquisitionFile updateAcquisitionFile)
```

Comment:
based on the name of this function I'd be tempted to call this something like:
addNoteIfFileStatusChanged

as is this appears to be something that checks if the file status has changed, not a function that adds a note if the file status has changed.